javascript/swiggy.js

```diff
@@ -1,52 +1,108 @@
 const body = document.body;
```

Comment:
Read about the following topics

Revealing modular pattern
Separation of concerns
Single responsibility principle
SOLID principles
DRY